lib/server-starter.js

```diff
@@ -44,6 +44,7 @@ class Server extends EventEmitter {
     this._process = undefined;
     this._exitHandlers = [];
     this._exit = undefined;
+    this._useFdPass = process.env.MOJO_SERVER_STARTER_AVOID_FDPASS ? false : process.platform !== 'win32';
```

Comment:
No Mojo specific code.

Reply:
And let the users decide if they want to use a file descriptor or not. Hidden magic only makes it harder to write portable code.